### PR TITLE
[13.0][UPD] Eficent -> ForgeFlow

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Update company name.

### CODE REVIEW

File: mail_debrand/README.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Mail Debrand
     :target: https://runbot.odoo-community.org/runbot/205/13.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| 
+|badge1| |badge2| |badge3| |badge4|
```
Comment (pedrobaeza) [inline mail_debrand/README.rst]: You shouldn't touch these files manually...

Comment (JordiBForgeFlow) [reply mail_debrand/README.rst]: @pedrobaeza agreed. But for all versions? Perhaps in older versions the bot does not auto generate the readme. Not sure.

Comment (pedrobaeza) [reply mail_debrand/README.rst]: Yes, everywhere is updated. If edited manually, you must removed the automatic trim of trailing whitespaces in the IDE.

Comment (MiquelRForgeFlow) [reply mail_debrand/README.rst]: @pedrobaeza A new .rst file will be autogenerated independently if I touch the file or not. So, why bother if I touch it or not?

Comment (pedrobaeza) [reply mail_debrand/README.rst]: Because it generates a lot of diff noise.

Comment (MiquelRForgeFlow) [reply mail_debrand/README.rst]: The diff noise will exist independently of what I do. We can't avoid the noise.

**Review by HviorForgeFlow — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

File: mail_debrand/README.rst (hunk 1 above)
Comment (MiquelRForgeFlow) [reply mail_debrand/README.rst]: @pedrobaeza the diff noise was removed. could we merge this?

**Pull request merged**